Sources/Plasma/PubUtilLib/plVault/plVaultClientApi.cpp

```diff
@@ -504,14 +502,12 @@ static void VaultNodeFetched (
 
     // Add to global node table
     auto it = s_nodes.find(node->GetNodeId());
-    hsRef<RelVaultNode> globalNode;
     if (it == s_nodes.end()) {
-        globalNode.Steal(new RelVaultNode());
-        globalNode->SetNodeId_NoDirty(node->GetNodeId());
-        it = s_nodes.emplace(node->GetNodeId(), globalNode).first;
-    } else {
-        globalNode = it->second;
+        hsRef<RelVaultNode> newGlobalNode(new RelVaultNode());
```

Comment:
This will cause the node to leak - a new instance of `NetVaultNode` has a ref count of 1, and the `hsRef` constructor increments the reference counter.

Reply:
Forgot a `hsStealRef` here, oops